### PR TITLE
Regression fetchOrder Poloniex

### PR DESCRIPTION
**Pull request by adi90x**
Hello,
Correction of a regression on Poloniex Exchange fetchOrder when `since` and `limit` where add to fetchOrders

### CODE REVIEW

File: js/poloniex.js [hunk 1]
```diff
@@ -549,8 +549,8 @@ module.exports = class poloniex extends Exchange {
         return this.filterBySinceLimit (result, since, limit);
     }
 
-    async fetchOrder (id, symbol = undefined, params = {}) {
-        let orders = await this.fetchOrders (symbol, params);
+    async fetchOrder (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let orders = await this.fetchOrders (symbol, since, limit, params);
```
Comment (kroitor) [inline js/poloniex.js]: Hi! Unfortunately, we can't do that, it will break fetchOrder compatibility with other exchanges.

Comment (adi90x) [reply js/poloniex.js]: Hello,
Just calling fetchOrders(symbol, undefined, undefined, params)  would be acceptable ?

Comment (kroitor) [reply js/poloniex.js]: Yep!

Comment (adi90x) [reply js/poloniex.js]: OK gonna push a new commit using params.limit and params.since

**Comment by kroitor** [comment]
@adi90x almost there... there are a few caveats with this code, described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

1. We can't access assoc key-values with `params.since`. That only works with JS and doesn't work with Python or PHP. Therefore we use `params['since']`.
2. We can't access `params['since']` either, if that key is missing. While most JS coders are used to these types of implicit undefined constructs, it will throw a KeyError exception in Python and will raise a warning in PHP. Therefore we have to check if those keys are set before accessing those values.
3. We can't pass them on to fetchOrders, if we use them in fetchOrder – need to strip them away with this.omit (array, keys).

But don't worry, I'll do all the fixes and will merge it this time, and from there you will see how it should be done for it to be remain compatible across 3 languages. We hope to see more contributions from you. Thx!

**Comment by adi90x** [comment]
Probably due to same regression  : fetchOrderStatus is returning error

**Comment by kroitor** [comment]
@adi90x will for for that too! Thx for the notice!

**Pull request merged**